### PR TITLE
Try fix links

### PR DESCRIPTION
**Pull request by sofiahag**
OK, so the `target="_blank"`didn't work. 
Unfortunately the links are not available locally so I can't test the code, but they are available on stage, so I hope it was just lacking a space, and it works on stage 🤞

### CODE REVIEW

**Comment by sofiahag** [comment]
OK, it didn't fix it 😬

**Comment by sofiahag** [comment]
And now the links are GONE 😱 
OK, I'll hold the testing a bit and try to figure it out 😆

File: run_dir/static/js/links.js [hunk 1]
```diff
@@ -27,19 +27,19 @@ function load_links() {
...
-        '<span class="align-top"><a class="text-decoration-none" "'+link_href+'target="_blank"'+'>'+link['title']+'</a>'
+        '<span class="align-top"><a class="text-decoration-none" "'+link_href+' target="_blank"'+'>'+link['title']+'</a>'
         +'</td><td>'+link['desc']+'<small> &nbsp;<a class="text-decoration-none" href="mailto:'+link['email']+'">'+link['user']+'</a>'+
         ' - '+date.toDateString()+'</span></td></tr>');
       $("#existing_links").append('<div class="link_wrapper"> \
                                     <div class="container"> \
                                     <div class="row justify-content-center">  \
                                       <div class="col-1 pr-0"> \
-                                        <a class="float-right"'+link_href+'><i style="font-size:18px;" class="fa fa-'+link_icon[link['type']]+'"></i> \
+                                        <a class="float-right"'+link_href+' target="_blank"'+'><i style="font-size:18px;" class="fa fa-'+link_icon[link['type']]+'"></i> \
```
Comment (aanil) [inline run_dir/static/js/links.js:37]:
```suggestion
                                        <a class="float-right"'+link_href+' target="_blank"><i style="font-size:18px;" class="fa fa-'+link_icon[link['type']]+'"></i> \
```

File: run_dir/static/js/links.js [hunk 2]
```diff
@@ -27,19 +27,19 @@ function load_links() {
...
                                         </a> \
                                       </div> \
                                       <div class="col-9"> \
                                         <h4> \
-                                          <a class="text-decoration-none" "'+link_href+'>'+link['title']+'</a> \
+                                          <a class="text-decoration-none" "'+link_href+' target="_blank"''>'+link['title']+'</a> \
```
Comment (aanil) [inline run_dir/static/js/links.js:42]:
```suggestion
                                          <a class="text-decoration-none" "'+link_href+' target="_blank">'+link['title']+'</a> \
```

**Comment by aanil** [comment]
> OK, so the `target="_blank"`didn't work.

It works for me.

**Comment by sofiahag** [comment]
OK, easier to spot these mistakes when you can test locally I guess 😏 

Hm I can't seem to pull your changes to stage, but if it works for you, I can merge?

**Review by aanil — APPROVED** [review]
👍

**Comment by sofiahag** [comment]
Great, thank you!

**Pull request merged**